### PR TITLE
Kevinayres aws mp gsd

### PR DESCRIPTION
**Pull request by kevinayres**
Updates to Rancher AWS Getting Started Guide. Work complete and ready for PR.

### CODE REVIEW

**Comment by tlssuse** [comment]
Hi, @dariavladykina, may you please add this document PR to your queue for review?

**Comment by kevinayres** [comment]
updated authors

**Comment by tlssuse** [comment]
Thanks for making these changes, @kevinayres .
I'll join you in keeping an eye open for any suggestions or questions from @dariavladykina .

**Comment by tlssuse** [comment]
@dariavladykina :  A specific version of Rancher is not mentioned.  This is because the offering in the AWS Marketplace uses the latest stable release of Rancher.  This means that the product version will change frequently, even though the offering and the way it works (as described in this document) does not.  As a result, we deviated from our normal practice of identifying specific versions.

At the time of this writing, the specific version of Rancher is 2.6.10, but the next update (which should be something like, 2.7.x) would not affect the content of the current document.

I'm happy to hop on a call to discuss this further.

**Comment by kevinayres** [comment]
Updated authors, variables.

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+image::aws-mp-rancher-16.png[Rancher Setup information, scaledwidth="85%", align="center"]
+
+. Wait for AWS Policy Simulator to check IAM permissions, then click __Next__.
+//
++
+If an error occurs at this stage, then the role was not correctly applied to the instance.
++ 
+image::aws-mp-rancher-17.png[AWS Policy Simulator, scaledwidth="85%", align="center"]
+
+. Choose a region for deploying your {eks_long} management cluster, the click __Next__.
```
Comment (dariavladykina) [inline kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc:339]:
```suggestion
. Choose a region for deploying your {eks_long} management cluster, then click __Next__. 
```

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+image::aws-mp-rancher-18.png[AWS Region for EKS cluster, scaledwidth="85%", align="center"]
+
+. Choose the size of your {productname} management cluster, then click __Next__.
++ 
+image::aws-mp-rancher-19.png[Size of management cluster, scaledwidth="85%", align="center"]
++
+[TIP]
+====
+'SMALL' works for most initial deployments.
+the instance that run the Rancher cluster can be resized when needed.
```
Comment (dariavladykina) [inline kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc:350]:
```suggestion
The instance that runs the {productname} cluster can be resized when needed.
```

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 3]
```diff
@@ -0,0 +1,508 @@
...
+====
+
+. Enter the fully qualified domain name (FQDN) for your Rancher management server, then click __Next__.
```
Comment (dariavladykina) [inline kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc:353]:
```suggestion
. Enter the fully qualified domain name (FQDN) for your {productname} management server, then click __Next__.
```

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 4]
```diff
@@ -0,0 +1,508 @@
...
+//
++
+[NOTE]
+====
+The subdomain containing the FQDN must be preconfigured as an AWS Route53 hosted zone.
+====
+image::aws-mp-rancher-20.png[Rancher URL, scaledwidth="85%", align="center"]
+
+. Take note of the URL at which {productname} will be served, then click __Next__ to begin deployment of Rancher.
```
Comment (dariavladykina) [inline kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc:362]:
```suggestion
. Take note of the URL at which {productname} will be served, then click __Next__ to begin deployment of {productname}.
```

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 5]
```diff
@@ -0,0 +1,508 @@
...
+//
++
+When complete, review the provided information.
+In particular, be sure to copy the initial password for the {productname} UI.
++ 
+image::aws-mp-rancher-22.png[Congratulations - Rancher is ready, scaledwidth="85%", align="center"]
++
+[NOTE]
+====
+The __Clean resources__ function gives you a mechanism for removing your Rancher environment.
```
Comment (dariavladykina) [inline kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc:376]:
```suggestion
The __Clean resources__ function gives you a mechanism for removing your {productname} environment.
```

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 6]
```diff
@@ -0,0 +1,508 @@
...
+.. Enter 'RancherSetupEC2policy' for the __Name__ and, optionally, enter a __Description__.
+
+.. Review the policy summary, then click __Create policy__.
+
+
+. Create a service role.
+// Kevin this step needs to be fleshed out like the one above.
+.. In __IAM__, select __Roles__.
+
+.. Create a new Role named 'RancherSetupEC2role' with use case type 'EC2', and attach the policy 'RancherSetupEC2policy' to it.
```
Comment (dariavladykina) [inline kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc:243]:
```suggestion
.. Create a new Role named 'RancherSetupEC2role' with use case type 'EC2' and attach the policy 'RancherSetupEC2policy' to it.
```

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 7]
```diff
@@ -0,0 +1,508 @@
...
+. When prompted, enter and confirm a new password for the 'admin' user, accept the terms and conditions, then click __Continue__.
++ 
+image::aws-mp-rancher-24.png[Rancher new password, scaledwidth="85%", align="center"]
+
+. You can now open your browser to the Rancher UI with the FQDN you provided during setup.
+For example, https://rancher.example.com.
++
+image::aws-mp-rancher-25.png[Rancher UI - Welcome, scaledwidth="85%", align="center"]
+
+. Terminate the temporary instance you created for the deployment automation to improve security and save cost.
```
Comment (dariavladykina) [inline kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc:405]:
```suggestion
. Terminate the temporary instance you created for the deployment automation to improve security and save costs. 
```

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 8]
```diff
@@ -0,0 +1,508 @@
...
+image::aws-mp-rancher-25.png[Rancher UI - Welcome, scaledwidth="85%", align="center"]
+
+. Terminate the temporary instance you created for the deployment automation to improve security and save cost. 
++ 
+image::aws-mp-rancher-30.png[AWS Terminate Instance, scaledwidth="85%", align="center"]
+
+. You can now create a 'Cloud Credential'  
++
+image::aws-mp-rancher-31.png[Rancher - Create Cloud Credential, scaledwidth="75%", align="center"]
+and deploy or import downstream clusters and node-groups for workloads.
```
Comment (dariavladykina) [inline kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc:412]:
```suggestion
and deploy or import downstream clusters and node groups for workloads. 
```

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 9]
```diff
@@ -0,0 +1,508 @@
...
+image::aws-mp-rancher-34.png[Rancher Priority Support offerings, scaledwidth="85%", align="center"]
+
+You can also purchase {productname2} via your normal AWS or SUSE sales channel or by contacting aws@suse.com directly to request a private offer.
+
+
+Once activated, a billing and metering container runs in your environment that validates the support entitlement you purchased through the {market} with the quantity of managed, downstream cluster nodes.
+This instantiates _{productname2}_ status for your deployment.
+
+
+If you procured {offering2} via the {market} you can get support by generating a 'supportconfig' and uploading it via the https://scc.suse.com/[SUSE Customer Center (SCC)].
```
Comment (dariavladykina) [inline kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc:440]:
```suggestion
If you procured {offering2} via the {market}, you can get support by generating a 'supportconfig' and uploading it via the https://scc.suse.com/[SUSE Customer Center (SCC)].
```

**Review by dariavladykina — COMMENTED** [review]
LVGTM! Just minor typos detected. Thanks!

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 10]
```diff
@@ -0,0 +1,508 @@
...
+
+=== Scope
+
+// = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+// Specify what this guide covers in no more than 2 sentences.
+// = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+
+In this guide, you learn how to install {productname} on {eks_long} using the _{offering1}_ listing in the {market}.
+
+From a single ephemeral appliance, you launch a three-node EKS cluster in a geography of your choosing, and install {productname} in this cluster using the current rancher-stable release.
```
[the remark this answers is not in the document]
Comment (tlssuse) [reply kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc]: Thank you.  I think this looks better.

File: kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc [hunk 11]
```diff
@@ -0,0 +1,508 @@
...
+
+=== Scope
+
+// = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+// Specify what this guide covers in no more than 2 sentences.
+// = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+
+In this guide, you learn how to install {productname} on {eks_long} using the _{offering1}_ listing in the {market}.
+
+From a single ephemeral appliance, you launch a three-node EKS cluster in a geography of your choosing and install {productname} in this cluster using the current rancher-stable release.
```
Comment (dariavladykina) [inline kubernetes/start/aws/adoc/gs_rancher_aws-marketplace.adoc:128]:
```suggestion
From a single ephemeral appliance, you launch a three-node EKS cluster in a geography of your choosing and install {productname} in this cluster using the current _rancher-stable_ release. 
```

**Comment by tlssuse** [comment]
@dariavladykina & @tomschr : This document is ready to merge.  If you concur, please do so and publish.

**Comment by tomschr** [comment]
Thanks @tlssuse! Looks fine, I'll merge it in a minute. :+1:

**Comment by tomschr** [comment]
@tlssuse Unfortunately, I just saw that I can't select the "Squash and merge" option for this repo. I think it would be really helpful to squash the commits to have a clean history. Otherwise we would end up with 50+ commits.

It can be enabled in the "Settings" tab under "General" -> "Pull Requests" and check the "Allow squash merging" box. Maybe that was on purpose, but I don't know.

Basically we have these options:

1. I could enable the option in the setting for you, or
2. Someone squashes the commits manually, or
3. Leave it as is and merge it.

I don't want to impose any workflow on you, that's why I'm asking. But in my humble opinion having a good history is a good thing.

What do you prefer?  :slightly_smiling_face:

**Comment by tlssuse** [comment]
Hi, @tomschr : Thank you.  I think we should squash the commits.  @kevinayres , since this is your fork, do you want to give it a try?

**Comment by tomschr** [comment]
Ahh sorry, I was a bit too fast. It's merged now. :see_no_evil:

**Comment by tomschr** [comment]
It's available now at https://docs.suse.com/trd/kubernetes/html/gs_rancher_aws-marketplace/index.html

**Pull request merged**